### ISSUE TITLE
[dev.icinga.com #6628] HA-Cluster Check Result Synchronisation

### ISSUE THREAD

**Issue by icinga-migration** [Icinga/icinga2#1755]
This issue has been migrated from Redmine: https://dev.icinga.com/issues/6628

**Created by jgruber on 2014-07-04 07:18:22 +00:00**

Assignee: _(none)_
Status: _Closed_  (closed on _2014-09-01 13:50:52 +00:00_)
Target Version: _(none)_
Last Update: _2014-09-01 13:50:52 +00:00 (in Redmine)_

    Icinga Version: 2.0.0-3

---

Check results in a HA-Cluster setup seem not be to synchronised between the nodes. You can only see the actual status of the check in the GUI on the node that executed the check. On the other node, you only see an old result or no result (pending).
Also posted on: http://www.monitoring-portal.org/wbb/index.php?page=Thread&threadID=31250

Examples:
Host Check on node1:
'Host';'Status';'Last\_Check';'Duration';'Attempt';'Status\_Information';
'krzicingat2';'UP';'06-30-2014 14:32:16';'2d 21h 51m 45s';'1/5';'PING OK - Packet loss = 0%, RTA = 0.36 ms';

Host Check on node2:
'Host';'Status';'Last\_Check';'Duration';'Attempt';'Status\_Information';
'krzicingat2';'PENDING';'N/A';'2d 21h 12m 58s';'1/5';'Host check scheduled for Fri Jun 27 17:19:20 CEST 2014';

Ping Service Check on node1:
'Host';'Service';'Status';'Last\_Check';'Duration';'Attempt';'Status\_Information';
'krzicingat2';'ping4';'OK';'06-27-2014 16:38:08';'2d 21h 56m 52s';'1/3';'PING OK - Packet loss = 0%, RTA = 0.56 ms';

Ping Service Check on node2:
'Host';'Service';'Status';'Last\_Check';'Duration';'Attempt';'Status\_Information';
'krzicingat2';'ping4';'OK';'06-30-2014 14:35:16';'2d 21h 16m 31s';'1/3';'PING OK - Packet loss = 0%, RTA = 0.87 ms';

---

**Relations**:

* relates [#6743](https://dev.icinga.com/issues/6743)
